index.rst

```diff
@@ -87,6 +87,57 @@ Catalog Data Access
 We assume that most user analysis cases that involve spatially localized searches of the Object or Source table, and/or the retrieval of ForcedSource data for limited sets of Objects, will go to Qserv via the TAP service.
 
 
+Bulk reprocessing of commissioning data
+---------------------------------------
+
+During Rubin/LSST commissioning it will be necessary to reprocess the data collected to date with evolving versions of the pipelines. 
+
+
+Processing of simulated images
+------------------------------
+
+Many analyses will benefit from being able to compare the LSST data products with the same analysis run on simulated images or catalogs. 
+
+
+Reprocessing of images with fake sources injected
+-------------------------------------------------
+
+Injecting fakes sources into real survey images and then reprocessing the images is an important technique to evaluate the performance of data reduction pipelines. 
```

Comment:
How much of the pipeline will need to be run to do this?  Do we go all the way back to the raw images and inject there?

Reply:
We start from PVIs I believe. not raws. I'll check and update

Reply:
I was just curious; this doesn't need to be addressed in this document.